grub-core/boot/i386/pc/boot.S

```diff
@@ -24,11 +24,14 @@
  *  defines for the code go here
  */
 
+#define TPM 1
```

Comment:
OK, though in the mean time seems there should be a TODO with each define or a single define one in a common header. No big deal though, few enough of them that they won't be missed in the future change I suppose.